.make/docker.mk

```diff
@@ -113,10 +113,18 @@ endif
 # This is a wildcard target to let you call any make target from the normal makefile
 # but it will run inside the docker container. This target will only get executed if
 # there's no specialized form available. For example if you call "make docker-start"
-# not this target gets executed but the "docker-start" target. 
+# not this target gets executed but the "docker-start" target.
 docker-%:
 	$(eval makecommand:=$(subst docker-,,$@))
 ifeq ($(strip $(shell docker ps -qa --filter "name=$(DOCKER_CONTAINER_NAME)" 2>/dev/null)),)
 	$(error No container name "$(DOCKER_CONTAINER_NAME)" exists to run the command "make $(makecommand)")
 endif
 	docker exec -t $(DOCKER_RUN_INTERACTIVE_SWITCH) "$(DOCKER_CONTAINER_NAME)" bash -ec 'make $(makecommand)'
+
+.PHONY: docker-golint
```

Comment:
I think this is not needed. Just run "make docker-golint". We have a whitespace target that executes all `docker-%` targets and inside the container.